src/lib/exporter/modernVideoExporter.ts

```
@@ -1414,6 +1414,10 @@ export class ModernVideoExporter {
 		if (this.audioProcessor) {
 			this.audioProcessor.cancel();
 		}
+		if (this.nativeH264Encoder) {
+			try { this.nativeH264Encoder.close(); } catch (_) {}
```

Biome (2.4.11)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)